### PR TITLE
sw/update-report-date-formats

### PR DESCRIPTION
**Pull request by SunainaWalia**
Raising this request to resolve https://dimagi-dev.atlassian.net/browse/QA-1299.
To update the date formats in these reports and make them consistent with the other reports. @mkangia @AmitPhulera, could you please review this?

Note, these static UCRs need to be updated both on India and TCL.

### CODE REVIEW

File: custom/icds_reports/ucr/reports/ls/ls_v2_report_child_names.json [hunk 1]
```diff
@@ -135,7 +135,10 @@
         "aggregation": "simple",
         "column_id": "dob",
         "field": "dob",
-        "transform": {},
+         "transform": {
+      "type": "date_format",
```
Comment (mkangia) [inline custom/icds_reports/ucr/reports/ls/ls_v2_report_child_names.json:139]: @SunainaWalia can you fix the indentation in these files?

Comment (SunainaWalia) [reply custom/icds_reports/ucr/reports/ls/ls_v2_report_child_names.json]: fixed the indentation @mkangia

**Review by mkangia — APPROVED** [review]
that is a weird date format
how do it appear on the UI?
`%-d/%-m/%Y`

**Comment by SunainaWalia** [comment]
> that is a weird date format

DD/M/YYYY example 15/5/2020

**Comment by mkangia** [comment]
Cool thanks @SunainaWalia . Mainly got confused with "-" in formats.

**Pull request merged**